rs/sns/governance/proto/ic_sns_governance/pb/v1/governance.proto

```diff
@@ -1488,14 +1489,15 @@ message Governance {
     // Name of the SNS project. This may differ from the name of the associated token.
     optional string name = 3;
 
-    // Description of the SNS project.
+    // Description of the SNS project
     optional string description = 4;
   }
 
   SnsMetadata sns_metadata = 21;
 
   // The initialization parameters used to spawn an SNS
   string sns_initialization_parameters = 22;
+  // (Q: WASM hashes??)
```

Comment:
Wasm module hash is the sha-256 sum of the installable canister module (since gzip-compressed Wasms are natively supported in ICP, the sha-256 sum is computed over the compressed module if it is being installed that way).